Adyen/UI/Form/Items/Address/FormFullAddressItem.swift

```diff
@@ -105,6 +111,18 @@ public final class FormAddressItem: FormValueItem<PostalAddress, AddressStyle>,
     
     // MARK: - Private
     
+    private func updateCountrySelectItem() {
+        guard let country = value.country, country != countrySelectItem.value.identifier else {
+            return
+        }
+        
+        guard let matchingElement = countrySelectItem.selectableValues.first(where: { $0.identifier == value.country }) else {
+            return // TODO: Log an error for the merchant
```

Comment:
@erenbesel how would we handle a case where a country is provided that is not "supported"? I'm looking for a way to indicate something is inconsistent now.